Realm/Tests/TestHost/main.m

```diff
@@ -8,7 +8,13 @@
 
 #import <TargetConditionals.h>
 
-#if TARGET_OS_IPHONE
+#if TARGET_OS_TV || TARGET_OS_WATCH
+
+// tvOS and watchOS don't support testing at this time.
```

Comment:
I believe those platforms do support testing now (previous versions did not), although we don't have device tests for those platforms yet.